contents/blog/product-360.md

```diff
@@ -0,0 +1,124 @@
+---
+date: 2022-01-18
+title: I used to think product people aren't needed
```

Comment:
Just thinking about punching this title up a bit. A few options:

1. I used to think product people aren't needed. I was wrong
2. How I learned I was wrong about product people
3. Why coders make the best product people

I'm inclined to (1). Just adding the bit on the end just adds a bit more bite, I think.

Reply:
Agreed with 1